examples/bioc/lib.rs

```diff
@@ -0,0 +1,157 @@
+//! # Basic Input Output Contract (BIOC)
+//!
+//! This is a Basic Input Output Contract (BIOC), which is intented to
```

Comment:
I think we need a better name. It is definitely about input output. But the point is that it is **not** basic. I think the mother of all contracts is still a good name. While it doesn't do any **useful** stuff it should still use all of ink!s features that are somehow observable off-chain. For example, storage and events are missing from this contract.